addons/token-exchange/dr_enablement.go

```diff
@@ -0,0 +1,46 @@
+package addons
+
+import (
+	"fmt"
+
+	replicationv1alpha1 "github.com/csi-addons/volume-replication-operator/api/v1alpha1"
+	ocsv1 "github.com/openshift/ocs-operator/api/v1"
+	multiclusterv1alpha1 "github.com/red-hat-storage/odf-multicluster-orchestrator/api/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/rest"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	RookCSIEnableKey              = "CSI_ENABLE_OMAP_GENERATOR"
+	OwnerMirrorPeerKey            = "multicluster.odf.openshift.io/owner-mirrorpeer"
+	MirroringModeKey              = "mirroringMode"
+	SchedulingIntervalKey         = "schedulingInterval"
+	ReplicationSecretNameKey      = "replication.storage.openshift.io/replication-secret-name"
+	ReplicationSecretNamespaceKey = "replication.storage.openshift.io/replication-secret-namespace"
+	Provisioner                   = "openshift-storage.rbd.csi.ceph.com"
+	HostNamePath                  = "/etc/hostname"
+)
```

Comment:
We don't need to export these. Keep it local to the  files that use it.

Reply:
removed the file